src/compiler/emitter.ts

```diff
@@ -4707,6 +4707,9 @@ if (typeof __param !== "function") __param = function (paramIndex, decorator) {
                 }
 
                 write("[\"require\", \"exports\"");
+                if (compilerOptions.noEmitExtends) {

```

Comment:
why not just assume a global will exist? and do you expect a module called `__extends` will exist?